Configuration/connectors/iam/mm2-cpc-iam-auth.json

```diff
@@ -5,6 +5,7 @@
     "source.cluster.alias": "msksource",
     "target.cluster.alias": "mskdest",
     "target.cluster.bootstrap.servers": "{TARGET CLUSTER BROKERS ADDRESS}",
+    "producer.override.bootstrap.servers": "{TARGET CLUSTER BROKERS ADDRESS}",
```

Comment:
I understand the purpose for this line, but in most recommended setup we don't need this override. I think it's good enough to explain it in the readme file.